pkg/registry/servicecatalog/server/server.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
+)
+
+var (
```

Comment:
@MHBauer few things:
- without using this Scheme the server startup was failing with errors described there https://github.com/kubernetes-incubator/service-catalog/pull/944#issuecomment-314353705
- in some places we now have to manually pass `Codecs` (didn't compile), and this part was copied from the latest sample-apiserver.
- Also see David's comment there: https://github.com/kubernetes-incubator/service-catalog/pull/944#discussion_r126947301